### PR TITLE
BZ#1778178 Required vCenter account privileges

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1778178

Preview: http://file.rdu.redhat.com/~choag/070820/vsphere-permissions-4.4/installing/installing_vsphere/installing-vsphere.html#installation-required-vcenter-account-privileges_installing-vsphere

### CODE REVIEW

**Comment by codyhoag** [comment]
@kalexand-rh As we discussed last month, I have taken the vCenter account privileges info that was approved/merged in the latest vSphere IPI 4.5 docs and am backporting it to previous versions, as was requested in the linked bug. Can you take a look?

**Comment by codyhoag** [comment]
@jinyunma can you confirm? Does this information apply for OCP 4.2-4.4? Thanks!

**Comment by codyhoag** [comment]
@jinyunma can you confirm this info for OCP 4.2-4.4.

**Comment by vikram-redhat** [comment]
@jinyunma are you still the right contact for this?

@xltian if this is not right, could you reassign?

**Comment by jinyunma** [comment]
@codyhoag sorry for the late, I use user with account privileges described in doc, server can be installed. So it looks good to me.

File: modules/installation-required-vcenter-account-privileges.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+A user requires the following privileges to install an {product-title} cluster:
+
+* Datastore
+** *Allocate space*
+* Folder
+** *Create folder*
+** *Delete folder*
+* vSphere Tagging
+** All privileges
```
Comment (jboxman) [inline modules/installation-required-vcenter-account-privileges.adoc:20]: Should this also be \*All privileges\*? Or is this a short hand for all listed privileges?

Comment (jboxman) [reply modules/installation-required-vcenter-account-privileges.adoc:20]: And if so, should it be _all listed privileges_ or something so it's clear that it refers to everything there. Or maybe it not being bold signifies that it isn't a UI element. If someone has the UI visible, it probably makes sense, but just reading it, I wasn't sure.

**Review by jboxman — APPROVED** [review]
Just one question. LGTM.

File: modules/installation-required-vcenter-account-privileges.adoc (hunk 1 above)
Comment (codyhoag) [reply modules/installation-required-vcenter-account-privileges.adoc:20]: The bold signifies specific permissions that must be selected. We're using _All privileges_ as a general term for all permissions available for that specific component.

**Pull request merged**